config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -163,6 +163,38 @@ presubmits:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-e2e-exp-main
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+  - name: pull-cluster-api-provider-azure-e2e-in-tree
```

Comment:
do we need a separate job for this specific flavor? Why not use the full e2e job like we currently do for external?

Reply:
So the "full" job just runs everything given that it has no skip config?

Reply:
however we should update the GINGKO_SKIP for jobs that currently skip the external cloud provider to instead skip the in tree flavor https://github.com/kubernetes/test-infra/pull/25043/files#diff-de63480688e0c1f8b4117747cd58cbafd15ea017e2ad279d178828b0b5097cd5R60

Reply:
If so then I suppose we don't need it. Is there any benefit of having the option to test this discretely? (sounds like we're not doing that for other targeted flavor tests, so probably not?)

Reply:
Ya! That's what this PR is for:

https://github.com/kubernetes/test-infra/pull/25041